### PR TITLE
Add com.github.kevinsmia1939.PySimpleCV

### PR DESCRIPTION
**Pull request by kevinsmia1939**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.github.kevinsmia1939.PySimpleCV.yaml [hunk 1]
```diff
@@ -0,0 +1,98 @@
+app-id: com.github.kevinsmia1939.PySimpleCV
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '22.08'
+command: PySimpleCV
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11 #PySimpleGUI Wayland not support
```
Comment (hfiguiere) [inline com.github.kevinsmia1939.PySimpleCV.yaml:8]:
```suggestion
  - --socket=x11 #PySimpleGUI Wayland not support
```

obviously you didn't try on a wayland session.

File: com.github.kevinsmia1939.PySimpleCV.yaml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        url: https://files.pythonhosted.org/packages/42/38/775b43da55fa7473015eddc9a819571517d9a271a9f8134f68fb9be2f212/numpy-1.23.5.tar.gz
+        sha256: 1b1766d6f397c18153d40015ddfc79ddb715cabadc04d2d228d4e5a8bc4ded1a
+        x-checker-data:
+          type: pypi
+          name: numpy
+
+  - name: pybind11
+    buildsystem: simple
+    build-commands:
+      - python3 -mpip install . --no-index --find-links=file://${PWD} --prefix=${FLATPAK_DEST}
```
Comment (hfiguiere) [inline com.github.kevinsmia1939.PySimpleCV.yaml:27]: are you sure? It should be either this or a `cmake-ninja` buildsystem. not both.

File: com.github.kevinsmia1939.PySimpleCV.yaml [hunk 3]
```diff
@@ -0,0 +1,98 @@
+app-id: com.github.kevinsmia1939.PySimpleCV
```
Comment (hfiguiere) [inline com.github.kevinsmia1939.PySimpleCV.yaml:1]: it should be `io.github`, this is now strictly enforced.

File: com.github.kevinsmia1939.PySimpleCV.yaml [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+          type: pypi
+          name: PySimpleGUI
+
+  - name: PySimpleCV
+    buildsystem: simple
+    build-commands:
+      - install -Dm744 PySimpleCV/PySimpleCV /app/bin
+      - install -Dm744 PySimpleCV/PySimpleCV_main_func.py /app/bin
+      - install -Dm644 data/${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      - install -Dm644 data/logo/logo_512.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
```
Comment (hfiguiere) [inline com.github.kevinsmia1939.PySimpleCV.yaml:88]: you could also install the SVG.

**Comment by hfiguiere** [comment]
I know it will fail at the linter stage:

bot, build com.github.kevinsmia1939.PySimpleCV

File: com.github.kevinsmia1939.PySimpleCV.yaml (hunk 3 above)
Comment (kevinsmia1939) [reply com.github.kevinsmia1939.PySimpleCV.yaml:1]: Is that ok, there is no io website for this project.

**Comment by kevinsmia1939** [comment]
Do anyone know if there is tkinter for python 3.10 for flatpak?
This seems to only use 3.9. Or does it not matter?
https://github.com/iwalton3/tkinter-standalone

**Comment by barthalion** [comment]
Can't you do something along `pip install tk`?

**Comment by kevinsmia1939** [comment]
> pip install tk

Hi, not really sure what you mean. The python-tkinter.yaml use `pip3 install` already. Also tk in pypi is TensorKit.

**Pull request closed without merging**